tools/patch-and-run-integration-test-suite.sh

```diff
@@ -16,7 +16,7 @@ function emit_error () {
 }
 
 JAVA_AGENT_JAR="${1}"
-HTTPSERVER_JAR="${2}"
+STANDALONE_JAR="${2}"
 PROJECT_ROOT_DIRECTORY=$(git rev-parse --show-toplevel)
 
 if [ "$#" -ne 2 ];
```

Comment:
In line 24: rename `httpserver` -> `standalone`.

Reply:
Fixed